symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkDatafeedConfig.java

```diff
@@ -11,17 +12,28 @@
 @API(status = API.Status.STABLE)
 public class BdkDatafeedConfig {
 
-    private String version = "v1";
-    private String idFilePath;
-    private BdkRetryConfig retry = new BdkRetryConfig(BdkRetryConfig.INFINITE_MAX_ATTEMPTS);
+  private String version = "v1";
+  private String idFilePath;
+  private BdkRetryConfig retry = new BdkRetryConfig(BdkRetryConfig.INFINITE_MAX_ATTEMPTS);
 
-    public String getIdFilePath() {
-        if (idFilePath == null || idFilePath.isEmpty()) {
-            return "." + File.separator;
-        }
-        if (!idFilePath.endsWith(File.separator)) {
-            return idFilePath + File.separator;
-        }
-        return idFilePath;
+  @Getter(AccessLevel.NONE)
+  private Boolean reuseDatafeedId;
+
+  public String getIdFilePath() {
+    if (idFilePath == null || idFilePath.isEmpty()) {
+      return "." + File.separator;
+    }
+    if (!idFilePath.endsWith(File.separator)) {
+      return idFilePath + File.separator;
+    }
+    return idFilePath;
+  }
+
+  public boolean isReuseDatafeedId() {
+    if (reuseDatafeedId == null) {
+      return true;
+    } else {
```

Comment:
we do not need "else" here we can return directly reuseDatafeedId.

Reply:
Well, as it is declared as `private Boolean reuseDatafeedId`, we still need this check to handle the case when the field hasn't been set in configuration. 

Maybe it would worth checking what happen if we declare `private boolean reuseDatafeedId = true;` (_including the default backward compatible value_). 

Otherwise I'd still have a preference for `return this.reuseDatafeedId == null ? true : this.reuseDatafeedId` :)

Reply:
`private boolean reuseDatafeedId = true;` works actually so I am going with that.